source/wp-content/themes/wporg-news-2021/sass/page/front-page/_latest-release.scss

```
@@ -70,18 +71,27 @@ body.news-front-page .front__latest-release {
 		@include break-medium() {
 			flex-grow: 1;
 			font-size: 90px;
+			padding:
+				calc(var(--wp--custom--alignment--edge-spacing) / 2)
+				var(--wp--custom--alignment--edge-spacing)
+				var(--wp--custom--alignment--edge-spacing)
+				var(--wp--custom--alignment--edge-spacing);
+
+			&:hover {
+				background-color: rgba(0, 0, 0, 0.1);
+
+				&::after {
```

I think this is meant to be on the `a`, not the heading tag, is that right? This might need to be moved below the `a` rules too, otherwise it'll be overridden by the later rule.

```suggestion
				a::after {
```

---

Good catch. Changing it to `a` restores the transform behavior, moving below does not appear to be necessary.